bemb/model/bemb.py

```diff
@@ -1196,6 +1244,7 @@ def reshape_observable(obs, name):
                 coef = (coef_sample_0 * coef_sample_1).sum(dim=-1)
 
                 additive_term = (coef * obs).sum(dim=-1)
+                additive_term *= term['sign']
```

Comment:
Can we just add one single `additive_term *= term['sign']` outside the `if-else-if` loop?

Reply:
yes, should be possible but i'll make sure